jhon_esteban/tarea_1/html/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/jhon_esteban/tarea_1/css/style.css">
```

Comment:
@Jhonec0814 , you should use relative paths, 

```suggestion
    <link rel="stylesheet" href="css/style.css">
```